type.proto

```diff
@@ -216,6 +227,14 @@ message Line {
     optional uint32 closing_time = 19;
 
     repeated Property properties = 21;
+    repeated GroupLink groups = 23;
+}
+
+message LineGroup {
+    optional string name = 3;
```

Comment:
You can start at one, a lot of our messages start a three probably because we had remove some old fields.